pkg/channel/consolidated/reconciler/controller/kafkachannel_test.go

```diff
@@ -334,8 +332,7 @@ func TestAllCases(t *testing.T) {
 			kafkaConfig: &KafkaConfig{
 				Brokers: []string{brokerName},
 			},
-			consumerGroupWatcher: NewConsumerGroupWatcher(ctx, &FakeClusterAdmin{}, 100*time.Millisecond),
-			kafkachannelLister:   listers.GetKafkaChannelLister(),
+			kafkachannelLister: listers.GetKafkaChannelLister(),
 			// TODO fix
```

Comment:
is this TODO still valid?

Reply:
I'm honestly unaware of the context of this TODO. It's been there from a much earlier change that used the Kafka Channel informer to enqueue on any update